gammapy/irf/psf_table.py

```diff
@@ -753,3 +759,414 @@ def _get_1d_table_psf(self, energy_index, **kwargs):
             self._table_psf_cache[energy_index] = table_psf
 
         return self._table_psf_cache[energy_index]
+
+class PSF3D(object):
+    """This class implements the format described here: :ref:`gadf:psf_table`.
```

Comment:
This intersphinx reference doesn't resolve and causes the docs build to fail:
https://travis-ci.org/gammapy/gammapy/jobs/139788155#L2134

I think this will work:
```
:ref:`gadf:psf-table`
```

(You have an underscore, not a minus:)
```
:ref:`gadf:psf_table`
```